pkg/controllers/user/alert/controller.go

```diff
@@ -161,7 +161,7 @@ func initClusterPreCanAlerts(clusterAlerts v3.ClusterAlertInterface, clusterName
 
 	eventRule := &v3.ClusterAlert{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: "clusteralert-deploment-event",
+			Name: "clusteralert-deployment-event",
```

Comment:
We can probably add a logic for deleting the old pre-caned event here. Not sure if the event is being referenced by any other objects in the system, and whether they have to be updated with the new reference too.